### PR TITLE
Cleanup codebase with Resharper

### PR DESCRIPTION
**Pull request by martijn00**
Fixes for https://github.com/MvvmCross/MvvmCross/issues/1922

### CODE REVIEW

File: MvvmCross-Plugins/DownloadCache/MvvmCross.Plugins.DownloadCache/MvxFileDownloadedEventArgs.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 namespace MvvmCross.Plugins.DownloadCache
 {
-    [Preserve(AllMembers = true)]
+    [All.Preserve(AllMembers = true)]
```
Comment (Cheesebaron) [inline MvvmCross-Plugins/DownloadCache/MvvmCross.Plugins.DownloadCache/MvxFileDownloadedEventArgs.cs]: Not too keen on all these Preserve attributes being prefixed like this

Comment (martijn00) [inline MvvmCross-Plugins/DownloadCache/MvvmCross.Plugins.DownloadCache/MvxFileDownloadedEventArgs.cs]: Somehow the compiler complains if we don't have them.

Comment (Cheesebaron) [inline MvvmCross-Plugins/DownloadCache/MvvmCross.Plugins.DownloadCache/MvxFileDownloadedEventArgs.cs]: Well... you did change the name space it was in.

**Comment by martijn00** [comment]
I'll look into the issue's and comment tomorrow.

**Pull request merged**